app/controllers/ChooseAnOptionController.scala

```diff
@@ -80,19 +82,26 @@ class ChooseAnOptionController @Inject()(resultsFileConnector: ResidencyStatusAP
   }
 
   def formattedExpiryDate(timestamp: Long): String = {
-    val expiryDate = new DateTime(timestamp).plusDays(3)
-    s"${expiryDate.toString("H:mma").toLowerCase()} on ${expiryDate.toString("EEEE d MMMM yyyy")}"
+    val zoneId = ZoneId.of("Europe/London")
```

Comment:
both the format funcs use some of the same code could pull out into its own function?
```
private def mapLongToZonedDateTime(timestamp: Long): ZonedDateTime =
     ZonedDateTime
      .ofInstant(
        Instant.ofEpochMilli(timestamp),
        ZoneId.systemDefault()
      )
```
Is it also possible to use `systemDefault`?

Reply:
The `formattedExpiryDate` func then becomes:

```scala annotate
  def formattedExpiryDate(timestamp: Long): String = {
    val expiryDate = mapLongToZonedDateTime(timestamp).plusDays(3)
    s"${expiryDate.format(DateTimeFormatter.ofPattern("h:mma")).toLowerCase} on ${expiryDate.format(DateTimeFormatter.ofPattern("EEEE d MMMM yyyy"))}"
  }
```